src/data/finset/sups.lean

```diff
@@ -118,7 +118,7 @@ section infs
 variables [semilattice_inf α] (s s₁ s₂ t t₁ t₂ u : finset α)
 
 /-- `s ⊼ t` is the finset of elements of the form `a ⊓ b` where `a ∈ s`, `b ∈ t`. -/
-def has_infs : has_infs (finset α) := ⟨image₂ (⊓)⟩
+protected def has_infs : has_infs (finset α) := ⟨image₂ (⊓)⟩
```

Comment:
I meant localize the notation, not the instances

Reply:
Sorry, what principal?

Reply:
Also, I'm still letting anyone do whatever with the notation. I'm just forcing it to be a binary operation with precedence 74.

Reply:
Can you elaborate on what you meant by this?

> Ah, but that goes against the general principle we adopted in pointwise actions.

Reply:
In `data.set.pointwise`, `data.finset.pointwise`, etc... we have definitions that are made instances in the locale `pointwise`, not notations that are declared in the locale `pointwise`.

Reply:
In that file we're using an existing notation though. I think a better example would be the `nhds` notation of topology, which isn't accessible without `open_locale topology`.

Reply:
But there is no `has_nhds` typeclass, so I'm not sure how the parallel holds.